src/sass/_silhouettes.scss

```diff
@@ -36,7 +36,7 @@
     fill: currentColor;
     stroke-dasharray: 1726;
     animation: homme 1.75s linear;
-    fill-opacity: 0;
+    fill-opacity: 1;
```

Comment:
Fill opacity should be modified in the 'homme' and 'femme' animation so that it only appears full after the line is drawn completely (dash-array/dash-offset animation).